content/docs/api-reference/invites-api/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Invites API"
+description: "Invites API Reference"
+lead: "Invites API Reference"
+date: 2021-06-09T11:56:23+01:00
+lastmod: 2021-06-09T11:56:23+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "api-reference"
+weight: 111
+toc: true
+---
+## Invites API Examples
+
+Invites can be used by a Root User to invite someone into the internal boundary of their tenancy to access assets and events.
```

Comment:
This is a bit of a mouthful? Can it be condensed at all? if not i think it may need some commas?